docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/csharp/precisionlist4.cs

```diff
@@ -1,13 +1,13 @@
-﻿// <Snippet7>
-using System;
+﻿using System;
 using System.IO;
 
 public class Example11
 {
     public static void Main()
     {
-        StreamWriter sw = new StreamWriter(@".\Doubles.dat");
-        Double[] values = { 2.2 / 1.01, 1.0 / 3, Math.PI };
+        // <Snippet7>
+        StreamWriter sw = new(@".\Doubles.dat");
```

Comment:
This won't work on linux or Mac:

```suggestion
        StreamWriter sw = new(@"./Doubles.dat");
```